### PR TITLE
Refactoring for better REST URL standards.

### PR DESCRIPTION
**Pull request by jgstorey**

### CODE REVIEW

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: backend/src/components/user.js [hunk 1]
```diff
@@ -207,17 +207,15 @@ function mapUserFacilityObjectForFront(data) {
...
-    log.verbose('getUserFacilities 1')
     let userFacilities = []
     let operation = `ofm_bceid_facilities?$expand=ofm_facility($select=accountnumber,address1_composite,name)&$filter=(statecode eq 0 and _ofm_bceid_value eq ${req.params.contactId}) and (ofm_facility/statecode eq 0)`
     if (onlyWithPortalAccess) {
       operation = operation + ` and (ofm_portal_access eq true)`
     }
-    log.verbose('getUserFacilities 2')
     const response = await getOperation(operation)
-    log.verbose('getUserFacilities 3')
+    log.verbose('getUserFacilities 1 response: ****************', response)
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply backend/src/components/user.js:216]: Removed logging.

File: backend/src/routes/user.js [hunk 2]
```diff
@@ -80,17 +80,11 @@ router.get(
...
-  [param('contactId', 'URL param: [contactId] is required').not().isEmpty()],
-  (req, res) => {
-    validationResult(req).throw()
-    const onlyWithPortalAccess = req.params.onlyWithPortalAccess === 'true'
-    return getUserFacilities(req, res, onlyWithPortalAccess)
-  },
-)
+router.get('/:contactId/facilities', passport.authenticate('jwt', { session: false }), isValidBackendToken, [param('contactId', 'URL param: [contactId] is required').not().isEmpty()], (req, res) => {
+  validationResult(req).throw()
+  const onlyWithPortalAccess = req.query.onlyWithPortalAccess === 'true'
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply backend/src/routes/user.js:85]: removed as const

File: frontend/src/components/settings/ManageUserDialog.vue [hunk 3]
```diff
@@ -258,7 +258,7 @@ export default {
      */
     async getUserFacilities(contactId, onlyWithPortalAccess) {
       try {
-        const res = await ApiService.apiAxios.get(ApiRoutes.USER_FACILITIES + '/' + onlyWithPortalAccess + '/' + contactId)
+        const res = await ApiService.apiAxios.get(ApiRoutes.USER_FACILITIES.replace(':contactId', contactId) + '/' + '?onlyWithPortalAccess=' + onlyWithPortalAccess)
```
[the remark this answers is not in the document]
Comment (jgstorey) [reply frontend/src/components/settings/ManageUserDialog.vue:261]: updated concatenated URL to template literal.

**Review by weskubo-cgi — APPROVED** [review]
Approved.

**Pull request merged**